src/js/index.js

```diff
@@ -81,6 +81,7 @@ var Grommet = {
   TBD: require('./components/TBD'),
   Icons: {
     Grommet: require('./components/icons/Grommet'),
+    Pulse: require('./components/icons/Pulse'),
```

Comment:
I always forget to add my new components to index.js, thanks for remembering to add it 👏